packages/plugin-flex/src/commands/flex/plugins/start.ts

```diff
@@ -4,6 +4,7 @@ import { FLAG_MULTI_PLUGINS } from 'flex-plugin-scripts/dist/scripts/pre-script-
 import { TwilioCliError, semver, env } from 'flex-dev-utils';
 import { readJsonFile } from 'flex-dev-utils/dist/fs';
 import { OutputFlags } from '@oclif/parser/lib/parse';
+import { PluginConfig } from 'flex-plugin-scripts/src/scripts/start';
```

Comment:
Remove this - this is to be moved into the webpack package and you wouldn't have access to it.